hermes/alertstream_handlers/ingest_from_hop.py

```diff
@@ -273,9 +273,8 @@ def handle_igwn_message(message: JSONBlob, metadata: Metadata):
             title=title,
             submitter=get_sender_from_metadata(metadata),
             authors='LVK',
-            data=alert,
             message_text='',
-            defaults={'published': published_time}
+            defaults={'published': published_time, 'data': alert}
```

Comment:
This is because I keep getting duplicate uuid errors in running readstreams locally because i think it is trying to ingest messages that are already in my system, but the data dict must be different since we append values into it on ingestion... That's my best guess at least.